development/nautobot_config.py

```diff
@@ -7,6 +7,8 @@
 from django.core.exceptions import ImproperlyConfigured
 from nautobot.core.settings import *  # noqa: F401,F403 pylint: disable=wildcard-import,unused-wildcard-import
 
+# pylint: disable=line-too-long
```

Comment:
Is this still needed given that you're setting `max-line-length=120` in the config?

Reply:
This is for lines which are greater than 120.

Reply:
Thanks. How many such lines are there? Can they be easily reformatted to fit the 120-character length instead?

Reply:
There were few,tried to resolve most of them there was one case wherein when I tried to break the line black tried to change it back.